.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,117 @@
+name: Feature request
+description: Report new data or fixing incorrect data from the [Whotracks.me](http://whotracks.me/) database
+labels: [pending triage]
+body:
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the issue
+      description: What would you like to change or add?
+      placeholder: Description
+    validations:
+      required: true
+
+  - type: input
+    id: url
+    attributes:
+      label: Where can it be repeated?
```

Comment:
Is this applicable?

Reply:
I thought this would be useful for reports where the user did not fill out additional company or tracker information

Reply:
We have a free form description field where the user can enter all the necessary information.